src/main/resources/lang/default.lang

```diff
@@ -2418,6 +2418,51 @@ experience cooldown change reasons:
 	plugin: plugin
 	pickup_orb: orb pickup, pickup orb
 
+banner patterns:
+	base: base banner pattern
+	border: border banner pattern, bordure banner pattern, bordure indented banner pattern
+	bricks: bricks banner pattern, field masoned banner pattern
+	circle: circle banner pattern
+	creeper: creeper banner pattern, creeper charged banner pattern
+	cross: cross banner pattern
+	curly_border: curly border banner pattern
+	diagonal_left: diagonal left banner pattern, left diagonal banner pattern
+	diagonal_right: diagonal right banner pattern, right diagonal banner pattern
+	diagonal_up_left: diagonal up left banner pattern, left up diagonal banner pattern, up left banner pattern
+	diagonal_up_right: diagonal up right banner pattern, right up diagonal banner pattern, up right diagonal banner pattern
+	flow: flow banner pattern
+	flower: flower banner pattern
+	globe: globe banner pattern
+	gradient: gradient banner pattern
+	gradient_up: gradient up banner pattern
+	guster: guster banner pattern
+	half_horizontal: half horizontal banner pattern, horizontal half banner pattern
+	half_horizontal_bottom: half horizontal bottom banner pattern, half bottom horizontal banner pattern, bottom half horizontal banner pattern, bottom horizontal half banner pattern, horizontal half bottom banner pattern, horizontal bottom half banner pattern
```

Comment:
do we really need all these combinations? I'm sure we can omit `half bottom horizontal`